libr/anal/state.c

```diff
@@ -51,25 +48,19 @@ R_API void r_anal_state_insert_bb(RAnalState* state, RAnalBlock *bb) {
 	if (!r_anal_state_search_bb (state, bb->addr) && state->current_fcn) {
 		r_list_append (state->current_fcn->bbs, bb);
 		state->bytes_consumed += state->current_bb->op_sz;
-		const char *key = sdb_fmt ("0x%08"PFMT64x, bb->addr);
-		if (!ht_insert (state->ht, key, bb)) {
+		if (!ht_up_insert (state->ht, bb->addr, bb)) {
```

Comment:
as long as now we have multiple type hashtables maybe good to add this `up` in the variable name holding the reference of the ht instance. So its easier for the user to know which kv types are expected to be used with each ht.

Reply:
we could, but do we want to? in general we don't specify the type in the name (e.g. we don't do "name_s", "bytes_ut64", etc.) and the fact that we have strongly typed hashtables (HtUP, HtPP and HtUU are very different types now) makes it impossible to mix them up.